src/components/settings-about.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+
+import {ScrollView, View} from 'react-native';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+
+import {Color, getColor} from '@app/colors';
+import {
+  ArrowForwardIcon,
+  DiscordIcon,
+  DocIcon,
+  GlobalIcon,
+  IconButton,
```

Comment:
Use alternative icons of the `Icon` component

Reply:
There is no alternative Discord Icon